src/init.cpp

```diff
@@ -354,7 +355,6 @@ void PrepareShutdown(NodeContext& node)
         }
         llmq::quorumSnapshotManager.reset();
         deterministicMNManager.reset();
-        evoDb.reset();
```

Comment:
can't make a suggestion via github (because it's a deleted line) but this should become `node.evodb.reset();` I think